_resources/kitodo/data/scripts/script_ocr_process_dir.sh

```diff
@@ -63,9 +63,12 @@ if test -n "$SCRIPT"; then
 fi
 
 if test -n "$WORKFLOW"; then
-    COMMAND+=" --workflow $WORKFLOW"
+    COMMAND+=" --workflow /workflows/$WORKFLOW"
 fi
 
+if test -n "$MQ_HOST" -a -n "$MQ_PORT"; then
+    COMMAND+=" --activemq-url tcp://$MQ_HOST:$MQ_PORT?closeAsync=false --activemq-queue TaskActionQueue --async true"
```

Comment:
Assuming you will rewrite the `--async` parameter:
```suggestion
    COMMAND+=" --activemq-url tcp://$MQ_HOST:$MQ_PORT?closeAsync=false --activemq-queue TaskActionQueue --async"
```